parachain/src/primitives.rs

```diff
@@ -198,9 +198,13 @@ impl From<i32> for Id {
 	}
 }
 
-const USER_INDEX_START: u32 = 1000;
+/// System parachain ids are `< 2000`
+const SYSTEM_INDEX_END: u32 = 1999;
+
+const USER_INDEX_START: u32 = 2000;
 const PUBLIC_INDEX_START: u32 = 2000;
```

Comment:
Since both `USER_INDEX_START` and `PUBLIC_INDEX_START` are `2000`, is there an actual difference between the two?